tests/integration/security/testdata/requestauthn/remotehttps.yaml.tmpl

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: security.istio.io/v1beta1
+kind: RequestAuthentication
+metadata:
+  name: request-authn
+spec:
+  selector:
+    matchLabels:
+      app: {{ .dst }}
+  jwtRules:
+  - issuer: "test-issuer-1@istio.io"
+    jwksUri: "https://jwt-server.{{ .Namespace }}.svc.cluster.local:8443/jwks"
+    outputPayloadToHeader: "x-test-payload"
```

Comment:
Could you change this to a different value (e.g. `x-test-payload-for-https-jwksuri) to make sure the jwt config is in effective for test? You should also update the test code with the new value.

Reply:
can you address the comment?